nxt/tokens.py

```diff
@@ -81,6 +96,8 @@ def get_standalone_tokens(raw_value, token_types=TOKENTYPE.ALL):
     else:
         if not raw_value or not isinstance(raw_value, str):
             return []
+    if token_types is None:
+        token_types = TOKENTYPE.ALL
```

Comment:
A code change that snuck in. I made this change in order to clean up the messy argument expansion sphinx was doing with `TOKENTYPE.ALL` as the default kwarg value. Not my favorite change ever.

Reply:
Hmmm its fine I guess. I know we use defaults like that in a few other places, I really don't want to have to change all those other signatures like this.